docs/WindowsBuild.md

```diff
@@ -1,408 +1,78 @@
-  -D ENABLE_TESTING=NO ^
-  -D dispatch_DIR=S:\b\2\cmake\modules ^
-  -G Ninja ^
-  -S S:\swift-corelibs-foundation
-
-ninja -C S:\b\3
-```
-
-- Add Foundation to your path:
-
-```cmd
-path S:\b\3\bin;%PATH%
-```
-
-## Build swift-corelibs-xctest
-
-```cmd
-cmake -B S:\b\4 ^
-  -D CMAKE_BUILD_TYPE=RelWithDebInfo ^
-  -D CMAKE_INSTALL_PREFIX=C:\Library\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr ^
-  -D CMAKE_MT=mt ^
-  -D CMAKE_Swift_COMPILER=S:/b/1/bin/swiftc.exe ^
-  -D dispatch_DIR=S:\b\2\cmake\modules ^
-  -D Foundation_DIR=S:\b\3\cmake\modules ^
-  -D LIT_COMMAND=S:\llvm-project\llvm\utils\lit\lit.py ^
-  -G Ninja ^
-  -S S:\swift-corelibs-xctest
-
-ninja -C S:\b\4
-```
-
-- Add XCTest to your path:
-
-```cmd
-
-path S:\b\4;%PATH%
-```
-
-## Test XCTest
-
-```cmd
-ninja -C S:\b\4 check-xctest
-```
-
-## Rebuild Foundation
-
-```cmd
-cmake -B S:\b\3 ^
-  -D CMAKE_BUILD_TYPE=RelWithDebInfo ^
-  -D CMAKE_INSTALL_PREFIX=C:\Library\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr ^
-  -D CMAKE_C_COMPILER=S:/b/1/bin/clang-cl.exe ^
-  -D CMAKE_MT=mt ^
-  -D CMAKE_Swift_COMPILER=S:/b/1/bin/swiftc.exe ^
-  -D CURL_LIBRARY="S:/Library/libcurl-development/usr/lib/libcurl.lib" ^
-  -D CURL_INCLUDE_DIR="S:/Library/libcurl-development/usr/include" ^
-  -D ICU_I18N_LIBRARY_RELEASE=S:\library\icu-67\usr\lib\icuin67.lib ^
-  -D ICU_ROOT=S:\Library\icu-67\usr ^
-  -D ICU_UC_LIBRARY_RELEASE=S:\Library\icu-67\usr\lib\icuuc67.lib ^
-  -D LIBXML2_LIBRARY=S:\Library\libxml2-development\usr\lib\libxml2s.lib ^
-  -D LIBXML2_INCLUDE_DIR=S:\Library\libxml2-development\usr\include\libxml2 ^
-  -D LIBXML2_DEFINITIONS="/DLIBXML_STATIC" ^
-  -D ENABLE_TESTING=YES ^
-  -D dispatch_DIR=S:\b\2\cmake\modules ^
-  -D XCTest_DIR=S:\b\4\cmake\modules ^
-  -G Ninja ^
-  -S S:\swift-corelibs-foundation
+The following [link](https://docs.microsoft.com/visualstudio/install/workload-component-id-vs-build-tools?view=vs-2022) helps in finding the component name given its ID for Visual Studio 2022.
 
-ninja -C S:\b\3
-```
-
-## Test Foundation
-
-```cmd
-ninja -C S:\b\3 test
-```
+> [!NOTE]
+> You only need the x64/x86 components when building for x64/x86 and vice versa. You need everything if you are cross-compiling.
 
-## Build swift-tools-core-support
+### Other dependencies
 
-```cmd
-cmake -B S:\b\5 ^
-  -D BUILD_SHARED_LIBS=YES ^
-  -D CMAKE_BUILD_TYPE=RelWithDebInfo ^
-  -D CMAKE_INSTALL_PREFIX=C:\Library\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr ^
-  -D CMAKE_C_COMPILER=S:/b/1/bin/clang-cl.exe ^
-  -D CMAKE_Swift_COMPILER=S:/b/1/bin/swiftc.exe ^
-  -D dispatch_DIR=S:\b\2\cmake\modules ^
-  -D Foundation_DIR=S:\b\3\cmake\modules ^
-  -D SQLite3_INCLUDE_DIR=S:\Library\sqlite-3.28.0\usr\include ^
-  -D SQLite3_LIBRARY=S:\Library\sqlite-3.28.0\usr\lib\SQLite3.lib ^
-  -G Ninja ^
-  -S S:\swift-tools-support-core
+In a PowerShell shell, run the following commands to install the rest of the required tools.
```

Comment:
This is definitely something that would work properly in cmd and PowerShell. I don't think that we need to specify a shell here.